demo/lib/topology2qemuopts.py

```diff
@@ -148,6 +174,8 @@ def dists(numalist):
         if "node-dist" in numaspec:
             for n in range(first_node_in_group, lastnode_in_group):
                 node_node_dist[n] = {int(nodename): value for nodename, value in numaspec["node-dist"].items()}
+    if lastnode_in_group == -1:
```

Comment:
nitnit: would it be safer/saner to do `< 0`

Reply:
Thanks, fixed!